### ISSUE TITLE
Should support all CSS math functions in <img sizes>

### ISSUE THREAD

**Issue by zcorpan** [validator/validator#803]
> A &lt;source-size-value> must not be negative, and must not use CSS functions other than the math functions.

https://html.spec.whatwg.org/multipage/images.html#sizes-attribute

https://drafts.csswg.org/css-values/#math

Test:
```html
<!doctype html>
<title>Test</title>
<img alt src="a.jpg" srcset="a.jpg 1000w, b.jpg 2000w" sizes="min(67em, 100vw)">
```

**Comment by zcorpan** [comment]
The change for math functions in `sizes` hasn't been implemented in any browser yet, though, it seems. :-(

https://github.com/whatwg/html/pull/3084#issuecomment-376243541

**Comment by sideshowbarker** [comment]
> The change for math functions in `sizes` hasn't been implemented in any browser yet, though, it seems. :-(
> 
> [whatwg/html#3084 (comment)](https://github.com/whatwg/html/pull/3084#issuecomment-376243541)

I that case, it seems like we want to hold off on adding checker support for those, right?

**Comment by zcorpan** [comment]
Yeah.

**Comment by sideshowbarker** [comment]
* Shipped in Chrome 79 https://bugs.chromium.org/p/chromium/issues/detail?id=825895
* Still open for Firefox https://bugzilla.mozilla.org/show_bug.cgi?id=1449070
* Still open for Safari https://bugs.webkit.org/show_bug.cgi?id=184027

So we still don’t yet have two engines with support
